### PR TITLE
prevent empty lines in address formatter

### PR DESCRIPTION
**Pull request by dominikager**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no

Will remove empty lines in the address

### CODE REVIEW

**Comment by dpfaffenbauer** [comment]
We are still supporting PHP 7.3, so the arrow function is a breaking change, please refactor it.

**Comment by dpfaffenbauer** [comment]
@dominikager can you please sign the CLA?

**Comment by dominikager** [comment]
@dominikager i just did, thanks

File: src/CoreShop/Component/Address/Formatter/AddressFormatter.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ public function __construct(TranslatorInterface $translator)
     /**
      * {@inheritdoc}
      */
-    public function formatAddress(AddressInterface $address, $asHtml = true)
+    public function formatAddress(AddressInterface $address, $asHtml = true): string
```
Comment (dpfaffenbauer) [inline src/CoreShop/Component/Address/Formatter/AddressFormatter.php:37]: please remove return type here, that is considered a BC break

**Comment by dpfaffenbauer** [comment]
@dominikager one small change please, than it is good to be merged

**Comment by dpfaffenbauer** [comment]
thanks

**Pull request merged**